lib/src/components/Avatar/avatar.module.scss

```diff
@@ -0,0 +1,76 @@
+@layer components {
+  .root {
+    flex-shrink: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: relative;
+    overflow: hidden;
+    aspect-ratio: 1 / 1;
+    border-radius: 50%;
+
+    width: var(--avatar-width);
+    height: var(--avatar-width);
+    font-size: calc(var(--avatar-width) / 2.5);
+
+    img {
+      object-fit: cover;
+      width: 100%;
+      height: 100%;
+    }
+
+    p {
+      font-weight: 500;
+      font-size: inherit;
+      color: var(--color-neutral-90);
+    }
+  }
+
+  .size {
+    &-sm {
+      --avatar-width: 20px;
```

Comment:
`--width or --avatarWidth or --avatar-witdth`
We need to found the best way to name this var

Reply:
My goto on this would be to mimic `styled-components` object style notation for css variables that don't exist in our heme.

styled-components object-styles notation is camelCase.

We would have something like :
`width: --avatarWidth`
`--avatarWidth: var(--according-width-from-theme)`

Does this make any sense to you: `camelCase for css variables not coming from theme` & `kebab-case` for those that come from it?